Chaincase.UI/ViewModels/SendViewModel.cs

```diff
@@ -404,6 +406,43 @@ public async Task<bool> SendTransaction(string password)
             return false;
         }
 
+        private IPayjoinClient? GetPayjoinClient()
+        {
+            if (!string.IsNullOrWhiteSpace(PayjoinEndPoint) &&
+                Uri.IsWellFormedUriString(PayjoinEndPoint, UriKind.Absolute))
+            {
+                var payjoinEndPointUri = new Uri(PayjoinEndPoint);
+                if (!_config.UseTor)
+                {
+                    if (payjoinEndPointUri.DnsSafeHost.EndsWith(".onion", StringComparison.OrdinalIgnoreCase))
+                    {
+                        Logger.LogWarning("PayJoin server is an onion service but Tor is disabled. Ignoring...");
+                        return null;
+                    }
+
+                    if (_config.Network == Network.Main && payjoinEndPointUri.Scheme != Uri.UriSchemeHttps)
+                    {
+                        Logger.LogWarning("PayJoin server is not exposed as an onion service nor https. Ignoring...");
+                        return null;
+                    }
+                }
```

Comment:
first commit references [Onion78 SendViewModel](https://github.com/armins88/Onion78/commit/6f3716147a621fbaf2219b0d202cd4864bbec55d)

I think these conditionals should be handled with exceptions instead of null returns

Reply:
oops, I was wrong. Exceptions here don't need to be handled, they actually do result in a null client, but they all need to be tested